docs/cloud-providers/add-cloud-providers-credentials.md

```diff
@@ -13,6 +13,16 @@ links:
     featured:
 ---
 
+# Generating Cloud Providers Credentials
+## GCP Service Account Credentials
+1. Access your GCP project or [create a new project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+2. Go to GCP IAM manager and select the `Service Accounts` option on the left menu
```

Comment:
```suggestion
2. Select the `Service Accounts` option under `IAM & Admin` on the navigation menu
```

Reply:
How about this:
https://github.com/devopness/devopness/blob/614e3070a24c883a787e519c4821679fe2bbfc67/docs/cloud-provider-credentials/get-gcp-service-account-credentials.md?plain=1#L29